lib/rdf/marmotta.rb

```diff
@@ -14,6 +14,14 @@ class Marmotta < ::RDF::Repository
       :ldpath => 'ldpath'
     }
 
+    CTYPES = RDF::Format.content_types.select do |key, values|
```

Comment:
Can we add some documentation here with the link to the issue we filed on Marmotta? If this is fixed I'd like us to know that we should clean this up later.